libs/features/branch-quick-links/branch-quick-links.js

```diff
@@ -0,0 +1,81 @@
+import { getMetadata, loadStyle, getConfig, createTag } from '../../utils/utils.js';
+
+function addLoader(elem, text) {
```

Comment:
You pass an `a` (anchorTag?) into this function, but then it turns into an `elem` ... I'd stick to naming this `a` as well or something more descriptive to what you actually pass in, unless it its _any_ element.